src/sentry/scim/endpoints/constants.py

```diff
@@ -48,6 +48,11 @@
     "detail": "Invalid Patch Operation.",
 }
 
+SCIM_429_RATE_LIMITED = {
+    "schemas": [SCIM_API_ERROR],
+    "detail": "You are being rate limited.",
```

Comment:
nit: This could be more specific- `You are being rate limited for too many invitations` is used elsewhere.